docs/resources/board.md

```diff
@@ -96,36 +96,39 @@ resource "honeycombio_board" "overview" {
 
 The following arguments are supported:
 
-* `name` - (Required) Name of the board.
-* `description` - (Optional) Description of the board. Supports markdown.
-* `column_layout` - (Optional) the number of columns to layout on the board, either `multi` (the default) or `single`. Only `visual` style boards (see below) have a column layout.
-* `style` - (Optional) How the board should be displayed in the UI, either `list` (the default) or `visual`.
-* `query` - (Optional) Zero or more configurations blocks (described below) with the queries of the board.
+-   `name` - (Required) Name of the board.
```

Comment:
This is a bit of a nit, and I'm sure your editor just did this on save, but none of the other files in the repo are formatted this way so if we could keep the `- ` for now that would be ideal